docs/exercise-201/exercise1.md

```diff
@@ -0,0 +1,106 @@
+### Configure Users permissions to access Argo CD
+
+#### Overview
+
+Argo CD provides a integration with many identity providers to authenticate and authorize users to interact with the UI and the API. In this exercise we are going to create a local user with no permission and gradually grant access to certain features.
+
+#### Exercises
+
+1. Local users are defined in the main Argo CD configmap (`argocd-cm`). Run the following command to create a new user (`workshop`) in Argo CD:
+
+```
+kubectl apply -f manifests/ArgoCD201-ArgoCDRBAC/argocd-cm.yaml
+```
+
+2. We need to login with Argo CD CLI so we can manage our newly created user. First we need to retrieve the admin password with:
+
+```
+kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
+```
+Then login with the following command:
+
+```
+argocd --port-forward --port-forward-namespace argocd login
+```
+For user enter: `admin`
+For password enter the string returned in the `kubectl` command above
+
+3. Now we need to change the password of the new user. The default password is the same as the one defined by Argo CD `admin` user. 
+
+Copy the same password used in exercise 2 and update the following command replacing the `ADMIN_PASSWORD` with it. Also update the new password replacing the `SOMETHING_SAFE` with something else (min 8 characters).
```

Comment:
Do you want to refer to  step2 instead of exercise 2

Reply:
good point. done